bedrock/firefox/redirects.py

```diff
@@ -504,4 +504,8 @@ def firefox_mobile_faq(request, *args, **kwargs):
 
     # bug 1283397
     redirect(r'^firefox/pocket/?$', 'https://getpocket.com/firefox/'),
+
+    # bug 1288552 - redirect /secondrun/ traffic from funnelcake test
+    redirect(r'^firefox(?:\/\d+\.\d+(?:\.\d+)?(?:a\d+)?)?/secondrun(?:/.*)?',
```

Comment:
It would be good to add some tests for this redirect. Also, is this permanent, or should we use a temporary redirect?